benches/mantis/jobs/mantis.ncl

```diff
@@ -5,69 +5,83 @@ let lib = import "../lib.ncl" in
 
 # Temporary dummy value
 let baseTags = [""] in
+let Params =
+  {
+    count
+      | number.Nat
+      | default
+      = 5,
+    role | [| `passive, `miner, `backup |],
+    datacenters | Dyn,
+    job | Dyn,
+    namespace | String,
+    logLevel | String,
+    mantisRev | String,
+    fqdn | String,
+    loggers | { _ : String },
+    network
+      | lib.contracts.OneOf [ "testnet-internal-nomad", "etc" ]
+      | default
+      = "testnet-internal-nomad",
+    networkConfig | String,
 
-let Params = {
-  count | number.Nat
-        | default = 5,
-  role | [| `passive, `miner, `backup |],
-  datacenters | Dyn,
-  job | Dyn,
-  namespace | String,
-  logLevel | String,
-  mantisRev | String,
-  fqdn | String,
-  loggers | {_: String},
-  network | lib.contracts.OneOf ["testnet-internal-nomad", "etc"]
-          | default = "testnet-internal-nomad",
-  networkConfig | String,
-
-  fastSync | Bool
-           | default = false,
-  reschedule | {..}
-               #{_ : lib.contracts.PseudoOr [
-               #         lib.contracts.OrableFromPred builtin.is_string,
-               #         lib.contracts.OrableFromPred builtin.is_number,
-               #         lib.contracts.OrableFromPred builtin.is_bool,
-               #         {
-               #           pred = builtin.is_record,
-               #           contract = {
-               #             attempts = 0,
-               #             unlimited = true,
-               #           },
-               #         }
-               #       ]
-               # },
-} in
-
+    fastSync
+      | Bool
+      | default
+      = false,
+    reschedule | { .. }
+    #{_ : lib.contracts.PseudoOr [
```

Comment:
Is it expected to re-indent comments like that? (I don't necessarily say it's a good or bad idea, but it's just surprising, so I'm genuinely wondering)

Reply:
Yes, comments will be affected by the indentation level. In the original, the source was being aligned to the annotation -- that seems to be common in the Mantis Nickel code -- but Topiary can't do that. As such, it treats it the same as any other code.